nixos/lib/make-channel.nix

```diff
@@ -4,17 +4,16 @@ pkgs.releaseTools.makeSourceTarball {
   src = nixpkgs;
   officialRelease = false; # FIXME: fix this in makeSourceTarball
   inherit version versionSuffix;
-  buildInputs = [ pkgs.nix ];
+  buildInputs = [ pkgs.nix (import ../../default2.nix) ];
```

Comment:
The tricks here is to depend on the channel so nix-env doesn't need to hit the networks